data_prep/dataset.py

```diff
@@ -7,6 +7,8 @@
 from data_prep.graph_utils import tf_idf_mtx, get_PMI
```

Comment:
So I assume you know that this file will be rewritten and you just used it for testing. I will use it for inspiration tho

Reply:
Exactly